guides/common/modules/proc_synchronizing-ansible-collections.adoc

```diff
@@ -49,7 +49,7 @@ endif::[]
 .. To sync {Project} from `console.redhat.com`, enter your token in the *Auth Token* field and enter your SSO URL in the the *Auth URL* field.
 ifdef::satellite[]
 +
-For more information, see https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.3/html/getting_started_with_automation_hub/proc-create-api-token[Retrieving your Red{nbsp}Hat Certified Collections Sync URL and API token] in _Managing Red{nbsp}Hat Certified and Ansible Galaxy collections in Automation Hub_.
+For more information, see https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.3/html/getting_started_with_automation_hub/[_Getting started with automation hub_].
```

Comment:
I believe the original link was pointing to https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.3/html/getting_started_with_automation_hub/con-create-api-token#proc-create-api-token.

Reply:
I suggest pointing to the whole guide, it will make for less fuss if they change something.